src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -59,6 +59,8 @@ public Config(final boolean verboose) {
         root = plugin.getDataFolder();
         lang = plugin.getConfig().getString("language");
 
+        final boolean isVirgin = Optional.ofNullable(root.listFiles(File::isFile)).map(files -> files.length == 0).orElse(true);
```

Comment:
Maybe use a name that more clearly communicates what it is about like `isInitialCreation` or similar?
```suggestion
        final boolean isInitialCreation = Optional.ofNullable(root.listFiles(File::isFile)).map(files -> files.length == 0).orElse(true);
```